### DISCUSSION TITLE
Synchronous meeting with rust-libp2p maintainer

### DISCUSSION THREAD

**Discussion by mxinden** [fleek-network/ursa#377] [General]
:wave: rust-libp2p maintainer here,

As mentioned before, great to see another rust-libp2p user, great to see another implementation of IPFS.

I would like to get to know you all and your mission with ursa. Maybe some of you are up for a synchronous call? If so, I am currently based in CET and not particularly busy next week.

**Comment by kckeiks** [comment]
Hi @mxinden, 

Thank you for reaching out!

I sent you an email to set up a time next week. Looking forward to meeting with you!
